docs/index.rst

```diff
@@ -1,49 +1,26 @@
 .. _index_gate:
 
+==========
 dwave-gate
 ==========
 
-.. include:: README.rst
-    :start-after: index-start-marker
-    :end-before: index-end-marker
-
-.. include:: README.rst
-    :start-after: example-start-marker
-    :end-before: example-end-marker
-
-Documentation
--------------
-
-.. sdk-start-marker
-
 .. toctree::
+    :caption: Reference documentation for dwave-gate:
     :maxdepth: 1
 
-    reference/index
-    release_notes
-
-.. sdk-end-marker
+    api_ref
 
-.. toctree::
-    :caption: Code
-    :maxdepth: 1
+About dwave-gate
+================
 
-    Source <https://github.com/dwavesystems/dwave-gate>
-    installation
-    license
-
-.. toctree::
-    :caption: D-Wave's Ocean Software
-    :maxdepth: 1
+.. include:: README.rst
+    :start-after: start_gate_about
+    :end-before: end_gate_about
 
-    Ocean Home <https://ocean.dwavesys.com/>
-    Ocean Documentation <https://docs.ocean.dwavesys.com>
-    Ocean Glossary <https://docs.ocean.dwavesys.com/en/stable/concepts/index.html>
+Usage Information
+=================
 
-.. toctree::
-    :caption: D-Wave
-    :maxdepth: 1
+.. todo:: add the intersphinx prefix to enable these links for self builds
 
-    D-Wave <https://www.dwavesys.com>
-    Leap <https://cloud.dwavesys.com/leap/>
-    D-Wave System Documentation <https://docs.dwavesys.com/docs/latest/index.html>
+*   :ref:`index_concepts` for terminology
+*   :ref:`qpu_gate_model_intro`
```

Comment:
This links incorrectly to https://docs.dwavequantum.com/en/latest/quantum_research/qpu_gate_model_intro.html#qpu-gate-model-intro (should instead be https://docs.dwavequantum.com/en/latest/quantum_research/gate_model_intro.html#qpu-gate-model-intro).

Reply:
Could you try deleting any previous build HTML in your local directory and rebuilding?
![image](https://github.com/user-attachments/assets/974eb469-ca34-47e8-97f4-acafee639ceb)